splinterd/sample_configs/splinterd.toml.example

```diff
@@ -128,3 +128,32 @@ oauth_client_secret = "<client-secret>"
 
 # Redirect URL for the OAuth provider used by the REST API
 oauth_redirect_url = "http://localhost:8080/oauth/callback"
+
+
+#Appenders is the structure and stdout is the items name
```

Comment:
Follow the conventions in the surrounding code -- put a space after the # marks.